tests/StackExchange.Redis.Tests/WrapperBaseTests.cs

```diff
@@ -130,7 +130,14 @@ public void HashSetAsync_2()
             wrapper.HashSetAsync("key", "hashField", "value", When.Exists, CommandFlags.None);
             mock.Verify(_ => _.HashSetAsync("prefix:key", "hashField", "value", When.Exists, CommandFlags.None));
         }
-
+        
+        [Fact]
+        public void HashStringLengthAsync()
+        {
+            wrapper.HashStringLengthAsync("key","field", CommandFlags.None);
+            mock.Verify(_ => _.HashStringLengthAsync("prefix:key", "field", CommandFlags.None));
+        }
```

Comment:
Can you please add the equivalent in DatabaseWrapperTests for non-async (you'll see the others mirrored there - e.g. below `HashSet_2`)? Happy to take as a TODO after the merge as well, just let me know :)

Reply:
Done